Gemfile

```
@@ -5,3 +5,8 @@ gemspec
 group :release do
   gem 'github_changelog_generator', :require => false
 end
+
+group :coverage do
```

On IRC I suggested to make it optional:
```suggestion
group :coverage, optional: !ENV['COVERAGE'] do
```